### PR TITLE
chore(website): fix link path

### PR DESCRIPTION
**Pull request by SunnyBoy-WYH**
subtask of https://github.com/apache/incubator-hugegraph-doc/issues/294

### CODE REVIEW

**Comment by SunnyBoy-WYH** [comment]
cc @imbajin  review

File: content/cn/docs/contribution-guidelines/validate-release.md [hunk 1]
```diff
@@ -154,26 +151,42 @@ bin/start-computer.sh -d local -r master
 ## 邮件模板
 
 检查完成后, 你应该按不同角色回复邮件: (普通开发者 & PMC 成员)
+```markdown
+[] +1 同意
 
+[] +0 没有意见
+
+[] -1 不同意，并注明原因
```
Comment (imbajin) [inline content/cn/docs/contribution-guidelines/validate-release.md:159]:
```suggestion
[] +1 approve

[] +0 no opinion

[] -1 disapprove with the reason
```

File: content/cn/docs/contribution-guidelines/validate-release.md [hunk 2]
```diff
@@ -154,26 +151,42 @@ bin/start-computer.sh -d local -r master
...
 I checked:
-1. All download links are valid
-2. Checksum and signature are OK
-3. LICENSE and NOTICE are exist
-4. Build successfully on macOS(Big Sur) 
-5. ....
+1.Download link/tag in mail are valid
+2.Checksum and GPG signatures are OK
+3.LICENSE & NOTICE & DISCLAIMER are exist
+4.Build successfully on Ubuntu22.04 & MacOS 14.2
```
Comment (imbajin) [inline content/cn/docs/contribution-guidelines/validate-release.md:167]:
```suggestion
4.Build successfully on XX OS version XXX
```

File: content/cn/docs/contribution-guidelines/validate-release.md [hunk 3]
```diff
@@ -154,26 +151,42 @@ bin/start-computer.sh -d local -r master
...
+5.No unexpected binary files
+6.Date is right in the NOTICE file
+7.Compile from source is fine under Java11
+8.No empty file & directory found
+9.Running server/loader/hubble process OK
```
Comment (imbajin) [inline content/cn/docs/contribution-guidelines/validate-release.md:172]:
```suggestion
9. Test running xxx service OK
```

File: content/cn/docs/contribution-guidelines/validate-release.md [hunk 4]
```diff
@@ -154,26 +151,42 @@ bin/start-computer.sh -d local -r master
...
-3. LICENSE and NOTICE are exist
-4. Build successfully on macOS(Big Sur) 
-5. ....
+1.Download link/tag in mail are valid
+2.Checksum and GPG signatures are OK
+3.LICENSE & NOTICE & DISCLAIMER are exist
+4.Build successfully on Ubuntu22.04 & MacOS 14.2
+5.No unexpected binary files
+6.Date is right in the NOTICE file
+7.Compile from source is fine under Java11
```
Comment (imbajin) [inline content/cn/docs/contribution-guidelines/validate-release.md:170]:
```suggestion
7.Compile from source is fine under JavaX
```

File: content/cn/docs/contribution-guidelines/validate-release.md [hunk 5]
```diff
@@ -154,26 +151,42 @@ bin/start-computer.sh -d local -r master
...
 I checked:
-1. All download links are valid
-2. Checksum and signature are OK
-3. LICENSE and NOTICE are exist
-4. Build successfully on macOS(Big Sur) 
-5. ....
+1.Download link/tag in mail are valid
+2.Checksum and GPG signatures are OK
+3.LICENSE & NOTICE & DISCLAIMER are exist
+4.Build successfully on Ubuntu22.04 & MacOS 14.2
```
Comment (imbajin) [inline content/cn/docs/contribution-guidelines/validate-release.md:184]:
```suggestion
4.Build successfully on XX OS Version XX
```

File: content/cn/docs/contribution-guidelines/validate-release.md [hunk 6]
```diff
@@ -154,26 +151,42 @@ bin/start-computer.sh -d local -r master
...
+5.No unexpected binary files
+6.Date is right in the NOTICE file
+7.Compile from source is fine under Java11
+8.No empty file & directory found
+9.Running server/loader/hubble process OK
```
Comment (imbajin) [inline content/cn/docs/contribution-guidelines/validate-release.md:189]:
```suggestion
9. Test running XXX service OK
```

File: content/en/docs/contribution-guidelines/validate-release.md [hunk 7]
```diff
@@ -67,7 +69,8 @@ gpg: key 28DCAED849C4180E: public key "coderzc (CODE SIGNING KEY) <zhaocong@apac
 gpg: Total number processed: 3
 gpg:               imported: 3
 
-# 2. Trust release users (trust n username mentioned in voting mail, if more than one user, just repeat the steps in turn or use the script below)
+# 2. Trust release users (trust n username mentioned in voting mail, if more than one user, 
+# just repeat the steps in turn or use the script below)
```
Comment (imbajin) [inline content/en/docs/contribution-guidelines/validate-release.md:73]:
```suggestion
# 2. Trust release users (trust n username mentioned in voting mail, if more than one user, 
#      just repeat the steps in turn or use the script below)
```

File: content/en/docs/contribution-guidelines/validate-release.md [hunk 8]
```diff
@@ -153,24 +153,43 @@ more reference official website: https://hugegraph.apache.org/cn/docs/quickstart
...
 I checked:
-1. All download links are valid
-2. Checksum and signature are OK
-3. LICENSE and NOTICE are exist
-4. Build successfully on macOS(Big Sur) 
-5. ....
+1.Download link/tag in mail are valid
+2.Checksum and GPG signatures are OK
+3.LICENSE & NOTICE & DISCLAIMER are exist
+4.Build successfully on Ubuntu22.04 & MacOS 14.2
```
Comment (imbajin) [inline content/en/docs/contribution-guidelines/validate-release.md:171]:
```suggestion
4.Build successfully on XX OS & Version XX
```

File: content/en/docs/contribution-guidelines/validate-release.md [hunk 9]
```diff
@@ -153,24 +153,43 @@ more reference official website: https://hugegraph.apache.org/cn/docs/quickstart
...
+5.No unexpected binary files
+6.Date is right in the NOTICE file
+7.Compile from source is fine under Java11
```
Comment (imbajin) [inline content/en/docs/contribution-guidelines/validate-release.md:174]:
```suggestion
7.Compile from source is fine under JavaXX
```

File: content/en/docs/contribution-guidelines/validate-release.md [hunk 10]
```diff
@@ -153,24 +153,43 @@ more reference official website: https://hugegraph.apache.org/cn/docs/quickstart
...
+8.No empty file & directory found
+9.Running server/loader/hubble process OK
```
Comment (imbajin) [inline content/en/docs/contribution-guidelines/validate-release.md:176]:
```suggestion
9. Test running XXX service OK
```

File: content/en/docs/contribution-guidelines/validate-release.md [hunk 11]
```diff
@@ -153,24 +153,43 @@ more reference official website: https://hugegraph.apache.org/cn/docs/quickstart
...
 I checked:
-1. All download links are valid
-2. Checksum and signature are OK
-3. LICENSE and NOTICE are exist
-4. Build successfully on macOS(Big Sur) 
-5. ....
+1.Download link/tag in mail are valid
+2.Checksum and GPG signatures are OK
+3.LICENSE & NOTICE & DISCLAIMER are exist
+4.Build successfully on Ubuntu22.04 & MacOS 14.2
```
Comment (imbajin) [inline content/en/docs/contribution-guidelines/validate-release.md:187]:
```suggestion
4.Build successfully on XX OS & Version XX
```

File: content/en/docs/contribution-guidelines/validate-release.md [hunk 12]
```diff
@@ -153,24 +153,43 @@ more reference official website: https://hugegraph.apache.org/cn/docs/quickstart
...
+5.No unexpected binary files
+6.Date is right in the NOTICE file
+7.Compile from source is fine under Java11
```
Comment (imbajin) [inline content/en/docs/contribution-guidelines/validate-release.md:190]:
```suggestion
7.Compile from source is fine under JavaXX
```

File: content/en/docs/contribution-guidelines/validate-release.md [hunk 13]
```diff
@@ -153,24 +153,43 @@ more reference official website: https://hugegraph.apache.org/cn/docs/quickstart
...
+8.No empty file & directory found
+9.Running server/loader/hubble process OK
```
Comment (imbajin) [inline content/en/docs/contribution-guidelines/validate-release.md:192]:
```suggestion
9. Test running XX process OK
```

File: content/cn/docs/contribution-guidelines/validate-release.md [hunk 14]
```diff
@@ -154,26 +151,42 @@ bin/start-computer.sh -d local -r master
...
-3. LICENSE and NOTICE are exist
-4. Build successfully on macOS(Big Sur) 
-5. ....
+1.Download link/tag in mail are valid
+2.Checksum and GPG signatures are OK
+3.LICENSE & NOTICE & DISCLAIMER are exist
+4.Build successfully on Ubuntu22.04 & MacOS 14.2
+5.No unexpected binary files
+6.Date is right in the NOTICE file
+7.Compile from source is fine under Java11
```
Comment (imbajin) [inline content/cn/docs/contribution-guidelines/validate-release.md:187]:
```suggestion
7.Compile from source is fine under JavaXX
```

**Review by imbajin — APPROVED** [review]

**Pull request merged**